tests/framework/integration.go

```diff
@@ -261,17 +272,85 @@ func (c integrationClient) TimeToLive(ctx context.Context, id clientv3.LeaseID,
 	return c.Client.TimeToLive(ctx, id, leaseOpts...)
 }
 
+func (c integrationClient) LeaseList(ctx context.Context) (*clientv3.LeaseLeasesResponse, error) {
```

Comment:
Any reason you re-declare those functions?

Reply:
It was added before Leases related PR is merged in upstream. I will rebase on latest code changes and send a new revision again. Thanks for pointing it out!!